fa17-ic13.csv

```diff
@@ -0,0 +1,102 @@
+MongoDB shell version v3.4.10,,,,,,,
```

Comment:
If sensitive, remove from *history*. If not, delete. If important, archive in UP data storage.